.github/workflows/release-please.yml

```diff
@@ -1,30 +1,18 @@
 name: Release-Please
 
-# triggered when tests complete on main branch
+# triggered when a PR is merged into main
 on:
-  workflow_run:
-    workflows:
-      - "Test"
-    branches:
-      - main
+  pull_request:
     types:
-      - completed
+      - closed
 
 jobs:
-  refused:
-    # Do not release if tests failed
-    runs-on: ubuntu-latest
-    if: ${{ github.event.workflow_run.conclusion == 'failure'}}
-    steps:
-      - id: refused_step
-        run: exit 1
-
   release-please:
+    if: github.event.pull_request.merged == true
```

Comment:
Here, I follow the syntax of https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-when-a-pull-request-merges